### DISCUSSION TITLE
Introduction to GitHub

### DISCUSSION THREAD

**Discussion by nidhi14jan** [skills/.github#622] [Introduction to GitHub]
As a new hubber as well as someone having no experience with the tool, this course was exactly what I wanted. I can now make sense of many of the terms I keep hearing and seeing in my inbox. Perfect intro to the tool.
The only negative feedback is that the page refresh after 20 secs did not take me to the next set of instructions all the time (specifically the merge pull request action and the last action after that).Not sure if its human error or a system error. I found a workaround though.
Thanks!

**Comment by GarciaMaM** [comment]
muy bien

**Comment by aflacake** [comment]
Congratulations, successfully completing it! 🎉✨
This is only a matter of input of the title (see instructions), for example `PROFILE.md` or even something else, don't write anything other than that. Not the description
